src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -6,6 +6,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import seedu.address.logic.commands.*;
```

Comment:
I am not sure, but I think this kind of import won't pass the Java CI.

Reply:
resolved!